docker/llm/finetune/lora/cpu/docker/bigdl-lora-finetuing-entrypoint.sh

```diff
@@ -1,46 +1,66 @@
 #!/bin/bash
 set -x
-source /opt/intel/oneapi/setvars.sh
-export CCL_WORKER_COUNT=$WORLD_SIZE
-export CCL_WORKER_AFFINITY=auto
-
-if [ "$WORKER_ROLE" = "launcher" ]
+if [ "$STANDALONE_DOCKER" = "TRUE" ]
 then
-  sed "s/:1/ /g" /etc/mpi/hostfile > /home/mpiuser/hostfile
-  export DATA_PATH="/ppml/data/$DATA_SUB_PATH"
-  sleep 10
+  source /opt/intel/oneapi/setvars.sh
+  export CCL_WORKER_COUNT=2
+  export CCL_WORKER_AFFINITY=auto
+  export MASTER_ADDR=172.18.0.3
   mpirun \
-    -n $WORLD_SIZE \
-    -ppn 1 \
-    -f /home/mpiuser/hostfile \
-    -iface eth0 \
-    -genv OMP_NUM_THREADS=$OMP_NUM_THREADS \
-    -genv KMP_AFFINITY="granularity=fine,none" \
-    -genv KMP_BLOCKTIME=1 \
-    -genv TF_ENABLE_ONEDNN_OPTS=1 \
-    python /ppml/lora_finetune.py \
-      --base_model '/ppml/model/'  \
-      --data_path "$DATA_PATH" \
-      --output_dir "/home/mpiuser/finetuned_model" \
-      --micro_batch_size $MICRO_BATCH_SIZE \
-      --bf16 > /home/mpiuser/launcher.log 2>&1
-  exit_status=$?
-  if [ $exit_status -ne 0 ];
+     -n 2 \
+     -ppn 2 \
+     -genv OMP_NUM_THREADS=24 \
+     -genv KMP_AFFINITY="granularity=fine,none" \
+     -genv KMP_BLOCKTIME=1 \
+     -genv TF_ENABLE_ONEDNN_OPTS=1 \
+     python /ppml/lora_finetune.py \
+       --base_model '/ppml/model/'  \
+       --data_path "/ppml/data/alpaca_data_cleaned_archive.json" \
+       --output_dir "/home/mpiuser/finetuned_model" \
+       --micro_batch_size 8 \
+       --bf16 > /home/mpiuser/launcher.log 2>&1
```

Comment:
remove `> /home/mpiuser/launcher.log 2>&1` to allow users to see outputs directly on screen